pkg/scheduler/framework/plugins/podtopologyspread/filtering.go

```diff
@@ -26,15 +26,18 @@ import (
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/client-go/util/workqueue"
 	"k8s.io/klog"
-	"k8s.io/kubernetes/pkg/scheduler/algorithm/predicates"
 	pluginhelper "k8s.io/kubernetes/pkg/scheduler/framework/plugins/helper"
-	"k8s.io/kubernetes/pkg/scheduler/framework/plugins/migration"
 	framework "k8s.io/kubernetes/pkg/scheduler/framework/v1alpha1"
 	"k8s.io/kubernetes/pkg/scheduler/nodeinfo"
 	schedulernodeinfo "k8s.io/kubernetes/pkg/scheduler/nodeinfo"
 )
 
-const preFilterStateKey = "PreFilter" + Name
+const (
+	preFilterStateKey = "PreFilter" + Name
+
+	// ErrReasonConstraintsNotMatch is used for EvenPodsSpread predicate error.
+	ErrReasonConstraintsNotMatch = "node(s) didn't match pod topology spread constraints"
```

Comment:
This has already been implemented in #86999

Reply:
And sorry for the previous mistake

Reply:
Sry didn't notice it previously. Let me update the pr to ignore changes in podtopologyspread.